6/ansible-6.0.0b1.deps

```diff
@@ -0,0 +1,103 @@
+_ansible_version: 6.0.0b1
+_ansible_core_version: 2.13.0
+amazon.aws: 3.2.0
+ansible.netcommon: 3.0.0
+ansible.posix: 1.4.0
+ansible.utils: 2.6.1
+ansible.windows: 1.10.0
+arista.eos: 5.0.0
+awx.awx: 21.0.0
+azure.azcollection: 1.12.0
+check_point.mgmt: 2.3.0
+chocolatey.chocolatey: 1.2.0
+cisco.aci: 2.2.0
+cisco.asa: 3.0.0
+cisco.dnac: 6.4.0
+cisco.intersight: 1.0.18
+cisco.ios: 3.0.0
+cisco.iosxr: 3.0.0
+cisco.ise: 2.4.1
+cisco.meraki: 2.6.2
+cisco.mso: 2.0.0
+cisco.nso: 1.0.3
+cisco.nxos: 3.0.0
+cisco.ucs: 1.8.0
+cloud.common: 2.1.1
+cloudscale_ch.cloud: 2.2.2
+community.aws: 3.2.1
+community.azure: 1.1.0
+community.ciscosmb: 1.0.5
+community.crypto: 2.3.1
+community.digitalocean: 1.19.0
+community.dns: 2.1.1
+community.docker: 2.6.0
+community.fortios: 1.0.0
+community.general: 5.0.0
+community.google: 1.0.0
+community.grafana: 1.4.0
+community.hashi_vault: 3.0.0
+community.hrobot: 1.3.0
+community.libvirt: 1.1.0
+community.mongodb: 1.4.0
+community.mysql: 3.2.1
+community.network: 3.3.0
```

Comment:
community.network is attempting to release a version 4.0.0 but couldn't make it in time: https://github.com/ansible-collections/community.network/pull/438

Let's wait until tomorrow to see if they can make it so it isn't locked to <4.0.0 until Ansible 7.